src/data/publications.ts

```diff
@@ -39,7 +39,7 @@ export interface Publication {
 export const PUBLICATIONS: Publication[] = [
   {
     title: 'ChoiceMates: Supporting Unfamiliar Online Decision-Making with Multi-Agent Conversational Interactions',
-    authors: ['Jeongeon Park', 'Bryan Min', 'Xiaojuan Ma', MEMBER.juhokim],
+    authors: ['Jeongeon Park', 'Bryan Min', 'Xiaojuan Ma', MEMBERS.juhokim],
```

Comment:
MEMBERS.juhokim sounds weird to me. Why not just MEMBER?

Reply:
`MEMBERS` is simply to denote that the object contains multiple Member objects. 
You're right in that `MEMBERS.juhokim` looks weird, but so does `MEMBER.juhokim`. 
How about instead we write `MEMBERS[juhokim]`?

Reply:
hmm... MEMBERS sounds like it is an array to me like how PUBLICATION is. If we go with MEMBERS, then we should change other data files consistently

Reply:
Yeah, I think we'll have to change the `MEMBER` data structure (eventually)
And also need to figure out a better way to retrieve and store publication data (particularly authors). 
I've been thinking of implementing a database and a retrieval API (e.g., https://gist.github.com/nichtich/97f06bbfa249f5d33e2d)